### ISSUE TITLE
Misleading subject line for memory hole

### ISSUE THREAD

**Issue by kwill** [mailpile/Mailpile#1954]
**User story:** I am a non-Mailpile user. I have become accustomed to getting Mailpile emails with the subject line "Encrypted message", so I assume any email with this subject line is encrypted (rather than checking the normal indicators).

**Suggestion:**

Change the subject line to something like "No subject", "Subject removed", "Subject embedded" or even "Enable [memory hole](#156) to read this subject".

**Comment by JocelynDelalande** [comment]
:+1: except if this subjet is a convention in memory hole (I haven't checked).

**Comment by BjarniRunar** [comment]
I am not sure I see the problem here.

Mailpile should never send a message with "Encrypted message" unless the message is actually encrypted. If it is sending a message with a subject that says it is encrypted but isn't, then that is a high priority bug.  Is that what you are saying?

If your client is capable of showing the memoryhole encrypted subject, and this somehow tricks you into thinking the message wasn't encrypted... how is that a problem? I can see there would be a problem if users were misled into thinking there was more security than was really the case, but if you get the idea that things were less secure than they actually were (and you can then go check the indicators to reassure yourself) then I don't see any harm here.

In short, I'm confused what the problem is and how your solution addresses it. :)

**Comment by kwill** [comment]
The problem is when a non-Mailpile user replies without changing the subject line, or forwards the message, etc. To a human being, the subject line remains "Encrypted mail" even when passed through a MUA that turns it into plaintext.

This is more a UX / security theatre / ecosystem issue than a technical error in Mailpile.

**Comment by BjarniRunar** [comment]
That is a good point - the subject line persisting even once the encryption has been stripped is a bad thing. So I agree, we should probably not generate a subject that becomes dangerously misleading down the line. We might also want to detect obvious subjects that are misleading and correct them at our end.

Thanks!
